core/src/main/java/uk/gov/justice/services/core/dispatcher/Dispatcher.java

```diff
@@ -38,7 +34,7 @@
      *
      * @param envelope the envelope to dispatch to a handler
      */
-    void asynchronousDispatch(final JsonEnvelope envelope) {
+    public void asynchronousDispatch(final JsonEnvelope envelope) {
```

Comment:
synchronousDispatch is now public, this can be reverted when SenderProducer is removed/refactored.